fixtures/attribute-behavior/AttributeTableSnapshot.md

```diff
@@ -10768,7 +10768,7 @@
 | `tabIndex=(string 'false')`| (initial)| `<number: -1>` |
 | `tabIndex=(string 'on')`| (initial)| `<number: -1>` |
 | `tabIndex=(string 'off')`| (initial)| `<number: -1>` |
-| `tabIndex=(symbol)`| (initial, warning)| `<number: -1>` |
+| `tabIndex=(symbol)`| (initial, warning, ssr error, ssr mismatch)| `<number: -1>` |
```

Comment:
The `ssr error, ssr mismatch` for Symbols doesn’t sound great, but it’s an existing issue with SVG attributes (you can search the file for many more matches). We should fix that separately.